SETUP/upgrade/13/20200215_alter_pm_default.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
`SETUP/upgrade/13` was for the R202002 release that just went out, create a new `SETUP/upgrade/14` directory and put this in there.